app/views/foods/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+  <h1>Foods list</h1>
+  <br>
+  <table class="table table-bordered">
+    <thead>
+      <tr>
+        <th>#</th>
+        <th>Food</th>
+        <th>measurement_unit</th>
+        <th>Price</th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+  
+    <tbody>
+      <% @foods.each_with_index do |food, index| %>
+        <tr>
+           <th scope="row"><%= index+1 %></th>
+           <td><%= food.name %></td>
+           <td><%= food.measurement_unit %></td>
+           <td><%= food.price %></td>
+           <td class="align-middle"><%= button_to 'Delete', "foods/#{food.id}", method: :delete, data: { confirm: 'Are you sure?' }, class: "btn-all" %></td>
+        </tr>
```

Comment:
- [ ] Please ensure that only the users that create foods can delete them. Currently, any user can delete any food even if they aren't the one that created it in the first place.